strapi/strapi
# Select permissions with the "Select all" command

### Why is it needed?
Bug description

🎬 Bug description
More than words → [Settings - API Tokens - 12 September 2022](https://www.loom.com/share/77bbe348ec2b405e9c671d5acac8743f)

### How to test it?

When “Select all” is selected after one or more specific permissions have been selected, I want all the permissions to be selected; then when “Select all” is unchecked, all the permissions should be unchecked.
